Automattic/jetpack
# Changelogger: Add New Prompt with Enquirer

## Proposed changes:

A redo of https://github.com/Automattic/jetpack/pull/34058 now that we have the enquirer package.

It adds a new question when adding a jetpack changelogger to check if site admins would need to know about it. If not, it gets assigned the `other` changelog type.

### Other information:


## Jetpack product discussion


pf4qpu-9d-p2
## Does this pull request change what data or activity we track or use?


no
## Testing instructions:






* Use `jetpack changelog add plugins/jetpack` and you should see the new prompt. If you said yes, it should then prompt you for the actual changelog type. If you say no, it should be skipped and the resulting changelog file should be "other."
* If you add a changelog to any other project, the changelogger should work per usual, without asking the question.
